index.html

```diff
@@ -193,5 +193,33 @@ <h1>About me</h1>
         </li>
     </div>
 </section>
+<div class="contact-form">
+    <h1>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+<form action="https://formspree.io/f/xbjedrnz" method="post">
+    <input type="text" id="name" name="name" maxlength="30" placeholder="Full name" required>
+    <input type="email" id="email" name="email" placeholder="Email address">
+    <textarea id="message" name="message" maxlength="500" rows="10" cols="30" placeholder="Enter text here"></textarea>
+    <button type="submit">Get in touch</button>
+</form>
+</div>
+<footer>
+    <ul>
+        <li><a href="#" class="github">
+             <img src="images/github 2.png" alt="github">
+         </a></li>
+         <li><a href="#" class="linkedin">
+             <img src="images/linkedin.png" alt="linkedin">
+         </a></li>
+         <li><a href="#" class="angel-list">
+             <img src="images/Vector 21.png" alt="angel-list">
+         </a></li>
+         <li><a href="#" class="twitter">
+             <img src="images/twitter 2.png" alt="Twitter">
+         </a></li>
+         <li><a href="#" class="medium">
+             <img src="images/medium 2.png" alt="Medium">
+         </a></li>
+     </ul>
```

Comment:
- [ ] [REQUIRED] The overall design is different from that provided in Figma Templates. Please ensure that you stick to the design templates in Figma as much as possible, this includes the font, colours, images and spacing (margin/padding). 
- [ ] I see that are using position fixed to layout your contact section. I believe that this styling method is causing problems now and will cause even more problems when you begin to implement the desktop version. I suggest you allow the elements to stack on top of each other in it natural flow- Read about CSS [box model here](https://blog.hubspot.com/website/css-box-model) for more tips. 

The images below show how your project is seen by your visitors. Some text is hiding when viewed at `375px`. 
From image `one` to `three`, there is text hiding on the right side. Please fix this.
**Your design**
One | two | three
--- | --- | ---
![image](https://user-images.githubusercontent.com/84156714/214801480-0cd708af-06d9-4eb2-aa47-137f44b76db6.png) | ![image](https://user-images.githubusercontent.com/84156714/214801613-bb44f529-819f-484f-99c2-ac28f2ddb8d8.png) | ![image](https://user-images.githubusercontent.com/84156714/214801672-5694d16e-05ff-4517-817f-d5251cf5e960.png)